image/cli/mascli/functions/pipeline_config_applications

```diff
@@ -152,6 +152,11 @@ function config_pipeline_applications() {
       prompt_for_input "Tablespace" MAS_APP_SETTINGS_TABLESPACE
       prompt_for_input "Indexspace" MAS_APP_SETTINGS_INDEXSPACE
     fi
+    echo
+    echo_h2 "Maximo IT Add-On"
+    echo -e "${COLOR_YELLOW} For information about your license, see https://ibm.biz/MAXIT81-License You are not required to accept the license terms unless you are installing Maximo IT add-on for Manage."
+    prompt_for_confirm_default_yes "Do you accept the license terms?" LICENSE_RESPONSE
```

Comment:
Is there not an option in the cli to install the IT addon? It would be nicer to have an "install Maximo IT addon" section and then display this prompt. We can then fail if no is selected.

Reply:
Hi @whitfiea from what I understand there will be an option to install Maximo IT add-on in the future. Right now I will make changes so the user has to accept both licenses.